docs/buildah-config.1.md

```diff
@@ -182,12 +182,16 @@ This option is typically only meaningful when the image's OS is Windows, and is
 typically set in Windows base images, so using this option is usually
 unnecessary.
 
-**--port**, **-p** *port*
+**--port**, **-p** *port/protocol*
 
 Add a *port* to expose when running containers based on any images which
 will be built using the specified container. Can be used multiple times.
-If *port* has a trailing `-`, and is already set, then the *port* is removed from the config.
-If the port is set to "-" then all exposed ports settings are removed from the config.
+To specify whether the port listens on TCP or UDP, use "port/protocol".
+The default is TCP if the protocol is not specified. To expose the port on both TCP/UDP,
+specify the port option multiple times. If *port* has a trailing `-` and is already set,
+then the *port* is removed from the configiuration. If the port is set to "-" then all exposed
```

Comment:
You have back tick here, and double quote above.  I think it should be backtick in both, but at least be consistent.
```suggestion
then the *port* is removed from the configiuration. If the port is set to `-` then all exposed
```